new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -0,0 +1,603 @@
+        .putDataItem(".dockerconfigjson", dockerConfigJson.getBytes());
+
+    boolean secretCreated = assertDoesNotThrow(() -> createSecret(repoSecret),
+        String.format("createSecret failed for %s", REPO_SECRET_NAME));
+    assertTrue(secretCreated, String.format("createSecret failed while creating secret %s in namespace",
+                  REPO_SECRET_NAME, opNamespace));
+
+    // map with secret
+    Map<String, Object> secretNameMap = new HashMap<String, Object>();
+    secretNameMap.put("name", REPO_SECRET_NAME);
+    // helm install parameters
+    opHelmParams = new HelmParams()
+        .releaseName(OPERATOR_RELEASE_NAME)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+
+    // Operator chart values to override
+    OperatorParams opParams =
+        new OperatorParams()
+            .helmParams(opHelmParams)
+            .image(operatorImage)
+            .imagePullSecrets(secretNameMap)
+            .domainNamespaces(Arrays.asList(domainNamespace))
+            .serviceAccount(serviceAccountName);
+
+    // install Operator
+    logger.info("Installing Operator in namespace {0}", opNamespace);
+    assertTrue(installOperator(opParams),
+        String.format("Operator install failed in namespace %s", opNamespace));
+    logger.info("Operator installed in namespace {0}", opNamespace);
+
+    // list helm releases matching Operator release name in operator namespace
+    logger.info("Checking Operator release {0} status in namespace {1}",
+        OPERATOR_RELEASE_NAME, opNamespace);
+    assertTrue(isHelmReleaseDeployed(OPERATOR_RELEASE_NAME, opNamespace),
+        String.format("Operator release %s is not in deployed status in namespace %s",
+            OPERATOR_RELEASE_NAME, opNamespace));
+    logger.info("Operator release {0} status is deployed in namespace {1}",
+        OPERATOR_RELEASE_NAME, opNamespace);
+
+    // check operator is running
+    logger.info("Check Operator pod is running in namespace {0}", opNamespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for operator to be running in namespace {0} "
+                    + "(elapsed time {1}ms, remaining time {2}ms)",
+                opNamespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(operatorIsRunning(opNamespace));
+
+  }
+
+  /**
+   *  Deploy a WebLogic domain with a defined configmap 
+   *  in configuration/model section of the domain resource.
+   *  The configmap has a sparse wdt model file that define a JDBC 
+   *  datasource targeted to the cluster
+   *  Once the WebLogic domain is up, verify the DataSource configuration 
+   *  using the RestAPI call thru adminserver public NodePort
+  */
+  @Test
+  @Order(1)
+  @DisplayName("Create model in image domain with a configmap")
+  @Slow
+  @MustNotRunInParallel
+  public void testCreateMiiDomain() {
```

Comment:
change test method name to testCreateMiiDomainWithConfigMap

Reply:
done